lib/ansible/modules/cloud/google/gcp_compute_address.py

```diff
@@ -0,0 +1,321 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017 Google
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+# ----------------------------------------------------------------------------
+#
+#     ***     AUTO GENERATED CODE    ***    AUTO GENERATED CODE     ***
+#
+# ----------------------------------------------------------------------------
+#
+#     This file is automatically generated by Magic Modules and manual
+#     changes will be clobbered when the file is regenerated.
+#
+#     Please read more about how to change this file at
+#     https://www.github.com/GoogleCloudPlatform/magic-modules
+#
+# ----------------------------------------------------------------------------
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+################################################################################
+# Documentation
+################################################################################
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ["preview"],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gcp_compute_address
+description:
+    - Represents an Address resource.
+      Each virtual machine instance has an ephemeral internal IP address and,
+      optionally, an external IP address. To communicate between instances on
+      the same network, you can use an instance's internal IP address. To
+      communicate with the Internet and instances outside of the same network,
+      you must specify the instance's external IP address.
+      Internal IP addresses are ephemeral and only belong to an instance for
```

Comment:
For a paragraph break (which seems like what this line is supposed to be) make the string a new list item.